Makefile

```diff
@@ -65,6 +63,12 @@ mypy-all:
 	# Remaining errors in tests:
 	mypy --config-file /dev/null raiden --ignore-missing-imports | grep error | wc -l
 
+linting-tools:
```

Comment:
I'd not duplicate the mypy call here but instead just depend on the existing mypy job

Reply:
The problem is I want to enforce the order:

flake8 -> mypy -> pylint. How can I do that?

Reply:
@ulope Sure I can do that. Just seemed like of an overkill to me, but since you spent the time to request it, you get it.